### DISCUSSION TITLE
Welcome to opentrafficsim Discussions!

### DISCUSSION THREAD

**Discussion by averbraeck** [averbraeck/opentrafficsim#1] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of the OpenTrafficSim (OTS) community, and to discuss among the developers what new features or changes in the software to implement. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas, and discuss possible new features of OTS.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community for the further developing the OTS simulation library we build together.

When the discussion leads to a new feature, these will be discussed in the 'Issues' section.

**Comment by ivanfborges** [comment]
Hello everybody
I downloaded the repository in my VSCode, but i'm having difficulty doing a simple simulation.
Followed the steps in the tutorial/readme, but unsuccessfully.
Can anyone guide me on how to play a simulation?
Thank you in advance
